### PR TITLE
Update usage docs - add packer init

### PR DESCRIPTION
**Pull request by sylviamoss**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1 @@
+* @hashicorp/packer
```
Comment (sylviamoss) [inline CODEOWNERS:1]: Adding us as codeowner to get notifications from this repo.

Comment (nywilken) [reply CODEOWNERS:1]: Good call!

File: packer/README.md [hunk 2]
```diff
@@ -39,27 +39,54 @@ debugging.
...
-    /mnt/test_docker_packer/template.json
-```
\ No newline at end of file
+docker run \
+    -v `pwd`:/workspace -w /workspace \
+    hashicorp/packer:latest \
+    build template.json
+```
+
+If plugins are necessary, we recommended to have the binaries inside a folder in the working directory and pass `-e PACKER_PLUGIN_PATH=/workspace/<wdir_plugin_path>` to the command above.
```
Comment (nywilken) [inline packer/README.md:82]: By plugins, do you mean third-party plugins not installed via Packer init? 

If so, what do you think about 
```suggestion
For the [manual installation](https://www.packer.io/docs/plugins#installing-plugins) of third-party plugins, we recommended that plugin binaries be placed a sub-directory under the working directory and adding `-e PACKER_PLUGIN_PATH=/workspace/<subdirectory_plugin_path>` to the command above.
```

**Review by nywilken — APPROVED** [review]
This looks good. Nice catch on the docker image not working with packer init. 

I left a couple of suggestions, but this is otherwise good to go.

File: packer/README.md (hunk 2 above)
Comment (sylviamoss) [reply packer/README.md:82]: Yes. But at some point I caught myself thinking if on 1.8.0 JSON users would have to install every plugin manually. T I remembered that this won't be necessary with Megan's PoC for importing plugins into packer.

File: packer/README.md [hunk 3]
```diff
@@ -39,27 +39,54 @@ debugging.
...
-    /mnt/test_docker_packer/template.json
-```
\ No newline at end of file
+docker run \
+    -v `pwd`:/workspace -w /workspace \
+    hashicorp/packer:latest \
+    build template.json
+```
+
+For the [manual installation](https://www.packer.io/docs/plugins#installing-plugins) of third-party plugins, we recommended that plugin binaries be placed a sub-directory under the working directory and adding `-e PACKER_PLUGIN_PATH=/workspace/<subdirectory_plugin_path>` to the command above.
```
Comment (sylviamoss) [inline packer/README.md:82]:
```suggestion
For the [manual installation](https://www.packer.io/docs/plugins#installing-plugins) of third-party plugins, we recommended that plugin binaries are placed under a sub-directory under the working directory. Add `-e PACKER_PLUGIN_PATH=/workspace/<subdirectory_plugin_path>` to the command above to tell Packer where the plugins are.
```

**Review by SwampDragons — APPROVED** [review]

**Pull request merged**